### DISCUSSION TITLE
Alethia coaching- integral unfoldment

### DISCUSSION THREAD

**Discussion by matthewmccarthy11** [life-itself/community#839] [Research]
Curious if anyone has any thoughts on this paper, which has a lot of interesting points, a few of which as they relate to DDS are:

1: self unfoldment opposed to self improvement (or development?) 
2: a 'generative ontology' (not descriptive) 

It comes from an organization called [ Alethia ](https://integralunfoldment.com/) 

[The Neuroscience of Enduring Transformation (V3-final) (1).pdf](https://github.com/life-itself/community/files/12786185/The.Neuroscience.of.Enduring.Transformation.V3-final.1.pdf)

**Comment by asimong** [comment]
On a very quick skim, this does look like an interesting attempt to bring together "parts" (like IFS), "process" "presence" and "nondual" work into a single frame. What the paper presents is a particular coaching method that has a sense of integrating these.

I see a small irony here in that different approaches suit different people, and even with approaches that intend to unify other approaches, the result of the attempted unification is always, inevitably, another approach. Naturally, any inventor of a new approach wants to bring it to a wider audience, and this seems like a great idea, in general. But also naturally inventors see their babies as special and wonderful. Thus, I see this paper as falling into the usual pattern of contrasting this new approach with less effective older approaches, and explaining why it is better. 

What I would find **really** interesting would be research that worked out a connection between the characteristics of clients and their histories, along with reasons why particular characteristics will be helped by particular coaching or therapeutic approaches. I'm sure that approaches incorporating "memory reconsolidation" will work well for many people. Who would not benefit? What are the alternative approaches or strategies?

**Comment by rufuspollock** [comment]
## Excerpts with commentary

### Different techniques work for different people and techniques vary widely

This seems incredibly similar to the major insight of Martin et al with the Finders program: a) there are a range of awakening techniques that vary substantially b) any given technique only works for some people (some of the time) c) you want to try lot of different techniques or even create a "cocktail" or "protocol" (or "ecology") of practices to be effective especially across people.

> I completed my initial coach training and certification in 2002. Like many coaches, I spent most of the next decade studying various developmental methodologies spanning both psychological and spiritual ranges of development. To me, a few things were striking that left me puzzled. First, all of the methods I learned were very different from each other. I naively expected that most self-development approaches would be variations on a theme. This is not what I found. **Second, I noticed that despite these differences, every method I learned was powerful and effective. However, third, I also noticed that each method also failed to work with certain clients. This puzzled me. All methods both work and also fail to work.** From this experience, I developed an eclectic coaching practice. I tried my best to assess the needs of each client and determine which method would fit them best. Sometimes I would mix and blend different methods in a single coaching engagement. This created mixed results, working sometimes and failing others. From my conversations with many other coaches, I learned that the journey from certification to eclectic practice was, by far, more the norm than the exception.

I'd also note that it demonstrates the lack of any organizing "map" for the coaching space - similar to the spiritual space (before Martin matrix etc)

### Then he finds a map 😄

> In 2012 I had an integrating insight that solved the puzzle I had about which method to use. I suddenly realized that the methods I knew each operated at different depths. Immediately I could see how these methods easily grouped into four depths. Each of the methods that I knew were specialists at one depth but generally ignored the others. When clients were functioning from the depth a method specialized in, it was powerful and effective. However, if the client was functioning from another depth, the same method was cumbersome or plain didn’t work. This insight helped to explain why all methods work and all methods fail to work. The crucial factor is depth.

- He finds a (rough) map i.e. "depths"
- Can assign "rafts" (i.e. practices) to the map (i.e. what depth they are working at)
- Has a way to map clients to depth
- Hence has a way to know which raft to which with which client

### His map has four levels: Parts, Process, Presence and Nonduality

Briefly, the Four Depths are defined as follows and listed in order from shallowest to deepest:

> 1. The Depth of Parts – Experience at this depth is dominated by the sense of separation. We experience ourselves as separate from other people and things. Inwardly, we experience many separate ego parts of ourselves (e.g., an ambitious part, a cautious part, a rejected part, a playful part, etc.) that have separate intentions, strategies, behaviors, and emotions (Schwartz & Sweezy, 2019). Each person has a unique configuration or constellation of ego parts that function as structurally consolidated emotional learnings. Ego parts can be understood as object relations (Kernberg, 1993), ego states (Watkins & Watkins, 1997), schemas (Young et al., 2003), or as winning strategies (Goss, 2015). These ego parts are the target for memory reconsolidation with experiential mismatches created by consciously accessing deeper depths of self- experience.
> 2. The Depth of Process – Experience at this depth is a flow of felt meaning that is accessed through bodily felt senses (Gendlin, 2007) and/or imaginal felt images (Corbin, 1976; Hillman, 1997; Bosnak, 2007). Felt senses and felt images reflect the subtleties of currently lived relatedness. Therefore, all process phenomena are unique to the moment. Accessing this depth creates many partial experiential mismatches with existing emotional learnings because what is arising freshly in the unfolding process of the present moment never exactly matches existing structurally consolidated emotional learnings, which exist at the Depth of Parts. Process phenomena are only accessible in real-time in the present moment.
> 3. The Depth of Presence – The hallmark of experience at this depth is the sense of innate wholeness and completeness. Presence is a field of awareness that can manifest as various virtuous qualities, for example, acceptance, contact, courage, equanimity, generosity, intelligence, intimacy, joy, kindness, love, passion, peace, perseverance, strength, trust, and value (Almaas, 2008). Accessing this depth usually creates complete experiential mismatches as the prevailing sense of ego deficiency is contrasted to the sense of innate capacity and sufficiency experienced at this depth. Like the Depth of Process, qualities of presence at this depth can only be accessed in the present moment within the sense of oneself as a field of conscious awareness.
> 4. The Depth of Nonduality – Experience at this depth is of non-separate and entirely unconditioned awareness (Fenner, 2007). In every way, experience at this depth contradicts and therefore mismatches with emotional learnings consolidated at the Depth of Parts where separation is experienced. This is the depth of profound spiritual realizations of unity consciousness and the domain of the deepest unfoldment of human development.

My quick reflection here would be that this is interesting ... and i suspect we're mixing waking up and cleaning up and growing up domains/lines quite a bit here (as is common). this is one of the biggest things i see happening in early stage theories - for obvious reasons: a line is simpler than a matrix which is simpler than a cube. That said a line is already better than no map at all.

### And the danger is we then get crude and likely distorted "neuroscience" to back up our simple map

Specifically i highly doubt that any given depth maps so nicely to EEG brain wave patterns.

> Largely speaking, mainstream culture only recognizes the Depth of Parts and remains unaware of deeper depths and therefore of the phenomenon of depth itself. However, these depths are empirically observable in various brain wave patterns using an electroencephalogram (EEG) (Wise, 2002). According to tests conducted using an EEG machine called the Mind Mirror 6.0:
> 
> 1. The Depth of Parts corresponds to high amplitude beta waves.
> 2. The Depth of Process corresponds to high amplitude theta waves.
> 3. The Depth of Presence corresponds to high amplitude and hemispherically synchronized gamma waves.
> 4. The Depth of Nonduality corresponds to high amplitude delta waves.
> 
> Alpha waves act to relax ego parts and form a bridge from the Depth of Parts into the Depth of Process.
> 
> Psychological and spiritual development is not a journey from the shallows to the depths. Rather, it is a journey of consciously inhabiting more and more of one’s depth. Initial EEG studies have indicated that Aletheia Coaching supports clients in cultivating what Anna Wise (2002) calls the “awakened mind” brain wave pattern which offers synergistic access to all four depths simultaneously.

### 4 rafts for 4 locations on the map

> Four Methods Integrated into One Method
> 
> With the Four Depths as an integrative framework, I began to experiment with ways of integrating the methods I had learned. To start with, I named exemplars for the development work at each depth from the methods that I knew, as follows:
> 
> 1. Parts Work – Internal Family Systems (IFS) therapy (Schwartz & Sweezy, 2019).
> 2. Process Work – Focusing (Gendlin, 2007).
> 3. Presence Work – The Diamond Approach1 (Almaas, 2008).
> 4. Nondual Work – Vajrayana Buddhism, especially Mahamudra (Brown, 2006) and contemporary non-affiliated nondual teachings (Fenner, 2007; Blackstone, 2007).

So here we have a nice ecology of practices. I again note that e.g. Martin incorporates, say, focusing into his protocol (not as an awakening practice per se but as a way to do cleaning to assist in awakening because trauma or other issues tend to be big blockers).

I also note we have a little bit of domains/lines confusion here e.g. levels 1 and 2 are more "cleaning up" i.e. shadow work. Whilst 3 and 4 (especially 4) are obviously much more into pure awakening.

### Self-improvements vs self-unfoldment

Basically: something is wrong here / missing vs i am already perfect and complete ... and maybe i can be more skilful.

> The provocative question posed in self-unfoldment coaching is “What if nothing is missing?” Naturally, this is not a question about the client’s skillfulness. Anyone can lack certain skills. I’m not very good, for example, at slam dunks in basketball. Instead, this is a question about the client’s wholeness as a human being. Very frequently, clients come to coaching with a sense of self-deficiency. Indeed, this often inclines them toward self-improvement which 1) doesn’t erase or edit existing emotional learning, 2) evokes the resistance of existing emotional learnings, 3) often creates relapses, and 4) which then reinforce the sense of self-deficiency. Self-unfoldment methods offer a powerful alternative.

NB: as a pure nomenclature ("branding"?) point i much prefer terms like cultivating being or unfolding over e.g. "self-development" or "self-improvement". (In fact i don't think i've ever used the term self-improvement). Overall i also prefer to keep "self" out of it as that itself is another trap. Unfortunately, a phrase like "being unfolding" sounds weird and awkward ... so my preference for now would be cultivating being or even cultivating becoming.

**Reply by asimong** [reply]
Thanks for this exploration @rufuspollock — lots of good stuff that is now covered in the replies. The only point I'd like to make right here (the only point I take exception to) is the idea that “depths” as a map is anything other than totally inadequate. A one-dimensional map??? This falls foul of similar critique of many many other attempts at linearising; measuring; single scale; etc. Yes, it's one person's simplistically-named model of a complex field.

I'm not at all critical of attempts to map out this field of enquiry. But I do think that each individual can have their own interpretation around what is “deeper” than what else. Can we try models that are clearly framed and introduced as multi-dimensional? Maybe there is a multi-dimensional model here, but I'm put off by the “depth” image.

**Comment by rufuspollock** [comment]
@matthewmccarthy11 really interesting and a great example of well documented and motivated "map plus rafts" aka "ontological framework" plus "ecology of practices".

**Comment by asimong** [comment]
Being prompted by the more careful reading by @rufuspollock I had a proper look through the example dialogue. Having read quite a lot about IFS, it strikes me that this is in fact just what good IFS practitioners do. March here seems to be introducing some extra theoretical constructs, specifically, experiential mismatch and memory reconsolidation, with the curiously added 5-hour “window” — I wonder what the empirical evidence is for such a window? I mean, it's perfectly plausible, but is it real?

So what I'm seeing here is a useful contribution in adding a new kind of compound perspective, which is interesting. But when he writes “Given the power and effectiveness of memory reconsolidation for generating sustained transformation, I predict that it will play a significant role in the future of coaching.” I see him as reifying a construct and writing as if it were some kind of objective phenomenon, rather than something that is all around us now. Otherwise I can't make sense of this statement as a prediction. Or does he mean something more like: “given the usefulness that I find of talking about this process using the conceptual framework of experiential mismatch and memory reconsolidation, I predict that other people will also find this framework helpful, and that this framework will come to dominate the discourse around coaching.” Which I guess is a rather over-optimistic claim.

**Comment by asimong** [comment]
@matthewmccarthy11 could you give us some more focused questions, digging a little deeper (and sticking your neck out a little further) than just asking for “any thoughts”? My sense is that ontological commoning starts when the questions get a little more focused, beyond people stating their initial viewpoints. Asking good questions here can really help to contribute to the dialogue.

**Comment by matthewmccarthy11** [comment]
100% @asimong, expanding on some initial points and a few questions

1: Does the distinction between a 'generative ontology' and a 'descriptive ontology' seem to be a meaningful one? (it is not explicitly outlined in the paper but I spoke briefly with Steve March and he used that distinction...you might have even brought this up a while ago @asimong but I'm not 100% sure) Generative in that it concerns 'enfoldment' or 'enfolding' or 'cultivating being' as it occurs IRL (always in different ways I suppose, even if there are patterns) and descriptive ontology dealing with the describing of models, values, and common language- or "map plus rafts"/ "ontological framework plus ecology of practices" @rufuspollock   A generative one then encapsulating the notion that 'being' can't be described, and there of course is a bit of an irony here- saying that being cannot be described, and then saying that it can be described with the term generative ontology- although I think the term can contain that paradox and that the term's main purpose is to contain that paradox. 

As it relates to what @asimong put in the ontological commoning write up:  "the ontology itself is about the concepts that are recognized in common, and the common language used to talk about those concepts, and the differences between different ontological perspectives." Can an ontology, though, involve processes which are not recognized in common- at least consciously, and these processes be understood as generative? Can an ontology also contain, for example, processes which are 'lived' and similarly encapsulated with the terms 'enfoldment' or 'being' or even 'self-organization'? An ontology (descriptive one) can never really 'express' the process in itself- but why would that not also be considered as part of the 'ontology' (understood as generative), the processes which occur in common, even if they are not apart of a descriptive model? 

It seems like OC is concerned with this more descriptive part, or how a group of people can come to agree on a descriptive ontology, but should ontological commoning refer to both? A 'generative ontology' in the sense that the 'system' itself arises and not only the language that is used to describe 'the system', or how we refer to the 'system'. (using the word 'system' lightly here to refer to the organized commoning that occurs) The distinction between speaking 'in the system' and speaking 'about the system' from Hofstadter might be interesting, where 'in the system' applies for generative (mostly), and 'about the system' applies for 'descriptive'

Which brings up the question: is a 'generative ontology' in the 'system', like for example a community, or in the 'lived experiences of people' in the community? More so a question to further contextualize the conversation, which I'm not sure myself, but maybe a 'generative ontology' exists in both to some degree

2: The idea of 'depths' is interesting, and I'm not really sure how to absorb it- if it is more so a metaphor to contextualize these practices, or if these different 'depths' have some real and actual dimension in human beings, which is backed up by 'the frequencies' they correspond to, as the paper suggests. If the latter is true- it seems like a really important point, but how exactly the 'frequencies' line up doesn't seem to be explained well.  I think you're probably right @rufuspollock, that it doesn't match so nicely

More on Ontological commoning- if you could just clarify or say y/n to the following points:

3: OC is (to put it in different words) a method of finding a shared language/set of concepts (ontology) which organize further activities (particularly when the context is complex/aims to solve a complex problem)
4: Can OC just be seen as 'human self-organization' (in regards to 'establishing some 'community' or 'social system')
5: An example of OC which also might be meaningful: education reform?
7: It does make sense to consider the psychological dimension (more so just agreeing)
8: I am not sure what you mean by 'technical systems'- also, does ontological commoning mostly refer to 'digital' spaces?

**Reply by asimong** [reply]
Reply to Point 1: 'generative ontology' and 'descriptive ontology' 

If you think this is significant, Matthew (and I guess you do) then maybe it would be worthwhile spelling out in a lot more detail exactly what a generative ontology might be, and whether it is different from – or more like a subclass of – descriptive ontologies.  I raise the possibility of it being a subclass, as to my mind there is a perfectly reasonable perspective that all ontologies are descriptive (of what exists and the relationships between what exists); and it could be that a generative ontology is one that describes in particular the potential in things — that would be, that takes potential as a fundamental ontological concept worthy of being represented as a first-class citizen of an ontology.

This raises all kinds of questions, to be sure, and to me it would be worth spelling out the questions it raises and possible attitudes towards those questions.

How do we see the relationship between my own concept of ontological commoning and ontologies as generative, or generative ontologies? I think perhaps best to answer this more fully in the new thread. But tl;dr: yes, I have a strong sense that ontological commoning is generative in possibly multiple ways, and this I feel is consistent with Silke Helfrich's insistence on prioritising the verbal form “commoning” over the nominal form “commons”

Reply to Point 2: depth

I fear that Steve Marsh is in danger of reducing a complex concept to a linear dimension, as implied by using the one-dimensional measurement word ”depth”. I really don't see this as having only one dimension, and for that reason alone I would steer well clear of that word, quite apart from its implied value scale of deeper = better (at least among our kind of people). I have seen similar critique of all developmental level theory. It is hard to factor out one clear useful dimension for level or depth. And yes, I agree, I have the same problem with “frequency” — as if we can reduce a wonderfully complex space to a single dimension. Talk about misleading metaphors! Frequency – and its correlate, wavelength – draw directly from wireless radio technology. Hard to think of anything more mechanistic and technology-centred!

**Comment by matthewmccarthy11** [comment]
In fact I am just going to make another quick thread as it relates to Ontological Commoning so the threads don't get too mixed- definitely feel free to reply here as it relates to the Alethia method (1-2) and these latter questions (3-8) in the other thread

**Reply by asimong** [reply]
Thanks Matthew @matthewmccarthy11 and yes, I'll do that.

And, maybe this is a good place to say this, thanks for these probing questions: very welcome; very stimulating — this is the kind of direction I feel we want to go as a research group. The more we explore down these avenues (some might dismiss them as rabbit-holes, but to me this would be missing some important potential) the more we are likely to come up with something that others may find of genuine interest, and therefore publishable.

**Reply by asimong** [reply]
See #844 (the new Ontological Commoning thread)
